.jvmopts

```diff
@@ -1,5 +1,4 @@
 -Xms2G
--Xmx2G
```

Comment:
ping @renatocaval 

I just want to make sure you're also aware of this change. I think `.jvmopts` should set a minimum memory requirement (`-Xms2G`) but I remove the maximum since developers with a powerful computer should be able to use all that power.

Reply:
You mean, someone with a 64G MacBook Pro?